src/components/Button/Button.styles.ts

```diff
@@ -37,7 +37,7 @@ position: relative;
   display: flex;
   gap: 12px;
   text-decoration: none !important;
-  height: ${(props) => heightMap[props.size]};
+  height: auto;
```

Comment:
Minor issue; it looks like the `line-height` is not quite right? There's quite a large space between lines of text that is wrapped in the screenshot posted.